specs/deposits.md

```diff
@@ -193,6 +193,9 @@ A predeployed contract on L2 at address `0x4200000000000000000000000000000000000
 certain block variables from the corresponding L1 block in storage, so that they may be accessed
 during the execution of the subsequent deposited transactions.
 
+A `seqnr` attribute is stored along with the L1 attributes to track the L2 block distance
```

Comment:
I really don't like `seqnr` as the name in the specs/contract. We don't need to limit identifiers to [6 characters](https://stackoverflow.com/questions/38035628/c-why-did-ansi-only-specify-six-characters-for-the-minimum-number-of-significa). In code it's a little better, but spelling out `sequenceNumber` is not that bad.